packages/wdio-browserstack-service/src/util.ts

```diff
@@ -17,6 +18,7 @@ import gitconfig from 'gitconfiglocal'
 import CrashReporter from './crash-reporter'
 import type { ITestCaseHookParameter } from './cucumber-types'
 import logPatcher from './logPatcher'
+import BrowserstackLauncherService from './launcher'
```

Comment:
Importing launcher from utils create a cyclic dependency

Reply:
similar has already been implemented in `CrashReporter`